api/generated/v1/types.pb.go

```diff
@@ -5,6 +5,7 @@ package v1api
 
 import (
 	fmt "fmt"
+	apiV1 "github.com/dell/csi-baremetal/api/v1"
```

Comment:
Does this changes comes from protobuf files ? How this file is generated?

Reply:
This is a mistake. I was hoping that multiple casts could be avoided